tests/unit/Collectors/CustomCollectorTest.php

```diff
@@ -47,7 +68,7 @@ public function testStartMeasure()
         $this->assertEquals(1, $this->collector->collect()->count());
         Assert::assertArraySubset([
             'label' => 'GET /endpoint',
-            'start' => 1000000.0,
+            'start' => 0.0,
```

Comment:
that's quite a change...

Reply:
`request_start_time` is now `1000`, and the span starts at the same time. That's why it went down to `0.0`.